src/interfaces/glossary.ts

```diff
@@ -27,6 +27,9 @@ export interface Term extends Element {
     type: 'term'
     index?: string
     reference?: string
+    name?: string
+    born?: number
+    died?: number
```

Comment:
Since these three attributes are only allowed when `index === "name"`, I'd rather express this as

```typescript
export interface NameTerm extends Term {
    index: 'name'
    name?: string
    born?: number
    died?: number
}
```

Although I'm not sure what would be a good name for this type.

Reply:
While I said that I'm not sure what would be a good name for this type, I think `TermForNameIndex` is a bit excessive :sweat_smile:.